docs/dunst.pod

```diff
@@ -264,7 +268,24 @@ Set to 0 to disable.
 A client can mark a notification as transient to bypass this setting and timeout
 anyway. Use a rule with 'set_transient = no' to disable this behavior.
 
-Note: this doesn't work on wayland yet.
+Note: this doesn't work on xwayland.
+
+=item B<layer> (Wayland only)
+
+One of bottom, top or overlay.
+
+Place dunst notifications on the selected layer. Using overlay
+will cause notifications to be displayed above fullscreen windows, though
+this may also occur at top depending on your compositor.
+
+The bottom layer is below all windows and above the background.
+
+Default: top
```

Comment:
Why not have overlay as the default layer? Wouldn't that be more appropriate for dunsts use, and more consistent with the X11 behavior?

I guess this also needs a note that notifications not shown during fullscreen will not be redisplayed like the current `pushback` behavior is.

Reply:
Yeah that's probably better. I'll change it